Applications/Custom/README.md

```diff
@@ -4,16 +4,22 @@ Applications inside this folder are dedicated to demonstrate how to create custo
 
 There are two ways to apply custom object to the code.
 
-1. Create an object as a part of client code and register it to NNTrainer on the client code.
+1. **Client**: Create an object as a part of client code and register it to NNTrainer on the client code.
   Easy to write, less portable
-2. Create an object as a dynamic library and NNTrainer load the dynamic library.
+2. **Plugin**: Create an object as a dynamic library and NNTrainer load the dynamic library.
   Portable, more configurations
 
+## List of Available Examples
 
-## Structure of the folder
+### Layer
 
-`*Client` demonstrates how to generate an object inside a client code and register on the fly.
-For example, `LayerClient` will demo about how to write a custom layer and register it.
+#### Related Folders:
 
-`*Plugin` demonstrates how to generate an object inside a plugin code and register on the fly.
-For example, `OptimizerPlugin` will demo about how to create a custom optimizer as a pluggable library.
+1. `LayerClients/` has a demo about how to write a custom layer and register it.
+2. `LayerPlugin/` contains test and build scripts to show how to build a custom layer and object as a seperate library(`.so`) and register it.
+
+#### Related Custom Objects:
+
+
+1. **Pow Layer**: A custom layer object which get x -> returns x^(exponent), exponent can be set by `layer.setProperty({"exponent={n}"});`.
```

Comment:
is it ok to delete pow layer and keep description?

Reply:
The deleted pow layer is duplicated one, that sneaked in 🤣

Reply:
haha ok.